ambry-frontend/src/main/java/com/github/ambry/frontend/GetAccountsHandler.java

```diff
@@ -152,5 +165,30 @@ private void start() {
       }
       return accounts;
     }
+
+    /**
+     * @return a pair of account and the container to get.
+     * @throws RestServiceException
+     */
+    private Pair<Account, Container> getContainer() throws RestServiceException {
```

Comment:
Why does this method return account along with container?  I saw you are setting the account id in the response, but can't you get that from container.getParentAccountId()?

Reply:
oh, that's true. Let me remove the account.